apps/console/src/features/organizations/components/edit-organization-role/organization-role-permission.tsx

```diff
@@ -99,15 +99,24 @@ export const OrganizationPermissionList: FunctionComponent<OrganizationPermissio
                 });
                 setPreviouslyCheckedKeys(previousFormCheckedKeys);
                 previouslyCheckedKeys?.forEach(key => {
-                    checkedNodes.push(getNodeByKey(key, permissions));
+                    const nodeByKey = getNodeByKey(key, permissions);
```

Comment:
Missing type/

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/0c9107c28e62fe5de005aee2a1e98e5f2af261be